content/docs/hooks-reference.md

```diff
@@ -6,11 +6,11 @@ prev: hooks-custom.html
 next: hooks-faq.html
 ---
 
-*Hooks* are an upcoming feature that lets you use state and other React features without writing a class. They're currently in React v16.8.0-alpha.1.
+*Hooks* son una característica que próximamente permitirá usar el estado y otras características de React sin escribir una clase. Están actualmente en React v16.8.0-alpha.1.
 
-This page describes the APIs for the built-in Hooks in React.
+Esta página describe las API para los Hooks incorporados en React.
```

Comment:
```suggestion
Esta página describe las APIs para los Hooks incorporados en React.
```

Reply:
The Academy prescribe not to pluralize acronyms in written, even if we do so in spoken language.
See [RAE consultas](http://www.rae.es/consultas/plural-de-las-siglas-las-ong-unos-dvd) and [Fundéu](https://www.fundeu.es/consulta/plural-de-abreviaturas-y-siglas-402/).